release_dashboard/lib/state/status_state.dart

```diff
@@ -26,7 +26,25 @@ class StatusState extends ChangeNotifier {
     await () async {
       releaseStatus = data;
     }();
+    notifyListeners();
```

Comment:
Missing `notifyListeners()` to update the release status, luckily this method wasn't used.